Signal/src/ViewControllers/MediaTileViewController.swift

```diff
@@ -111,9 +111,6 @@ public class MediaTileViewController: UICollectionViewController, MediaGalleryDa
         self.footerBarBottomConstraint = footerBar.autoPinEdge(toSuperviewEdge: .bottom, withInset: -kFooterBarHeight)
 
         updateSelectButton()
-
-        self.view.layoutIfNeeded()
-        scrollToBottom(animated: false)
```

Comment:
Perf boost: Scrolling to bottom triggers the "autoFetchMore" logic, which is unnecessary because we already ensure we've fetched the items around our "focusedItem" and scroll to that item when viewDidAppear.